packages/workbox-precaching/test/sw/revisioned-caching.js

```diff
@@ -118,48 +118,6 @@ describe('sw/revisioned-caching()', function() {
     });
   });
 
-  const invalidTypeRevisions = [
-    null,
-    undefined,
-    false,
-    true,
-    12345,
-    {},
-    [],
-  ];
-  invalidTypeRevisions.forEach((invalidRevision) => {
-    it(`should throw an errror for bad revision value '${JSON.stringify(invalidRevision)}'`, function() {
-      let caughtError;
-      try {
-        cacheManager.addToCacheList({revisionedFiles: [{url: VALID_PATH_REL, revision: invalidRevision}]});
-      } catch (err) {
-        caughtError = err;
-      }
-
-      if (!caughtError) {
-        throw new Error('Expected file manifest to cause an error.');
-      }
-      caughtError.message.indexOf(`The 'revision' parameter has the wrong type`).should.equal(0);
-    });
-  });
-
-  it(`should throw an errror for an empty string revision.`, function() {
```

Comment:
It was due to https://github.com/GoogleChrome/workbox/pull/664#discussion_r126388087, but I can reinstate it.